planetiler-core/src/main/java/com/onthegomap/planetiler/writer/TileArchiveMetadata.java

```diff
@@ -27,7 +27,7 @@ public MbtilesMetadata(Profile profile) {
     );
   }
 
-  public MbtilesMetadata(Profile profile, Arguments args) {
+  public TileArchiveMetadata(Profile profile, Arguments args) {
     this(
```

Comment:
looks like you need to update these to "tileset_" for the tests to pass. That's technically a breaking change since people might have scripts that set these - could you make then check `mbtiles_*` as a fallback?

Reply:
Instead of checking mbtiles_* as a fallback we could make planetiler explicitly exit until the user changes their configuration to `tileset_`? do we plan to keep the `mbtiles_*` around forever?

Reply:
changed to leave as `mbtiles_` for now and keeping this PR an internal refactor; can change this next time we bump the user-facing API